README.md

```diff
@@ -14,11 +14,25 @@ I'm attempting to make a self-contained AngularJS Directive which will allow you
 # Browser Support
 * IE10+, Android 3+, iOS 6+, basically all modern browsers!
 
+
+# Installation
+Get the library and its stylesheet using one of the following ways:
+
+1. **Github**
+ - [unminified] : [angular-image-crop.js](https://raw.githubusercontent.com/AshBardhan/angular-image-crop/master/dist/angular-image-crop.js) and [angular-image-crop.css](https://raw.githubusercontent.com/AshBardhan/angular-image-crop/master/dist/angular-image-crop.css)
+ - [minified] : [angular-image-crop.min.js](https://raw.githubusercontent.com/AshBardhan/angular-image-crop/master/dist/angular-image-crop.min.js) and [angular-image-crop.min.css](https://raw.githubusercontent.com/AshBardhan/angular-image-crop/master/dist/angular-image-crop.min.css)
```

Comment:
@andyshora Once you merge this code. You've to change the link from `https://raw.githubusercontent.com/AshBardhan/angular-image-crop/master/dist/angular-image-crop.min.js` to `https://raw.githubusercontent.com/andyshora/angular-image-crop/master/dist/angular-image-crop.min.js`

Similarly do this for other files.